app/controllers/devise_overrides/omniauth_callbacks_controller.rb

```diff
@@ -54,6 +54,9 @@ def get_resource_from_auth_hash # rubocop:disable Naming/AccessorMethodName
   end
 
   def validate_business_account?
+    # Only validate for Chatwoot Cloud instances
+    return true unless ChatwootApp.chatwoot_cloud?
+
     # return true if the user is a business account, false if it is a gmail account
     auth_hash['info']['email'].downcase.exclude?('@gmail.com')
   end
```

Comment:
Should we just move this to an installation config (a comma separated list of domains we want to block) and use this here?

Reply:
This is just for Google SSO. We already support using an installation config of blocked domains for regular signups.

Reply:
@sojan-official As discussed, let us unify the config and make it so that there is only config we need to update to block the domains by which a user can signup.